Sources/Argo/Types/StandardTypes.swift

```diff
@@ -194,40 +194,15 @@ public extension Collection where Iterator.Element: Decodable, Iterator.Element
 
     - returns: A decoded array of values
   */
-  static func decode(_ json: JSON) -> Decoded<[Iterator.Element]> {
+  public static func decode(_ json: JSON) -> Decoded<[Iterator.Element]> {
```

Comment:
`Decoded<[Element]>`? Could also be `Decoded<Array>`, which would implicitly have the same `Element` type as the extension.

Reply:
Oh right. This is an artifact from when this was an extension on `Collection`. I'll go with `Decoded<Array>`, I dig that.

Reply:
Made the same change for `Dictionary`.